tests/tests_integration/test_api/test_assets.py

```diff
@@ -279,24 +279,26 @@ def test_upsert_2_asset_one_preexisting(self, cognite_client: CogniteClient) ->
                 external_id=[new_asset.external_id, preexisting.external_id], ignore_unknown_ids=True
             )
 
-    def test_filter_on_metadata_key(
+    def test_advanced_filter_on_metadata_key(
```

Comment:
Let's not remove existing tests on `.filter`, we want to have those. Add new tests instead.

Reply:
Sure, I can do that. Was trying not to add to many integration tests as they take a while to run.